editor/src/main/com/mbrlabs/mundus/editor/ui/modules/Outline.kt

```diff
@@ -220,10 +220,43 @@ class Outline : VisTable(),
 
         })
 
-        // right click menu listener
-        tree.addListener(object : InputListener() {
+        tree.addListener(object : ClickListener() {
+            override fun clicked(event: InputEvent?, x: Float, y: Float) {
+                if (tapCount != 2)
+                    return
+
+                val node = tree.getNodeAt(y)
```

Comment:
I think you can use `val go = tree.getNodeAt(y)?.value;` and you can remove the next if statement.

Reply:
Yeah that worked, cleaned it up a bit